glide.yaml

```diff
@@ -31,3 +31,6 @@ import:
   version: ~3.0.0
 - package: github.com/rs/cors
   version: ~1.1.0
+- package: github.com/rs/cors
+  version: ~1.1.0
+- package: github.com/dustin/go-humanize
```

Comment:
to add this, first remove the manually added line, then use `glide get github.com/dustin/go-humanize`.  Then you'll have an updated glide.yaml and glide.lock